concent_api/core/validation.py

```diff
@@ -131,3 +135,9 @@ def validate_golem_message_signed_with_key(
                 public_key
             )
         )
+
+
+def validate_golem_message_subtask_results_rejected(subtask_results_rejected : message.tasks.SubtaskResultsRejected):
```

Comment:
The type annotation is wrong. This function handles types other than `SubtaskResultsRejected` too (by raising `Http400`).

Reply:
it is not. I have created this function, so I know what argument it is supposed to take ;)
It is for checking `SubtaskResultsRejected`

Reply:
It's a function that validates data received from the client. You cannot guarantee that the client will send you only `SubtaskResultsRejected` and nothing else. So it must handle other types too and the annotation is wrong (it states that the function takes only this type).

If you could assume that `subtask_results_rejected` is always of this type then this `if` would have to be an `assert`:

``` python
if not isinstance(subtask_results_rejected,  message.tasks.SubtaskResultsRejected):
    raise Http400("subtask_results_rejected should be of type:  SubtaskResultsRejected")
```